CRM/Civicase/Upgrader.php

```diff
@@ -408,66 +414,19 @@ public function upgrade_0001() {
   }
 
   /**
-   * Example: Run a couple simple queries.
-   *
-   * @return TRUE on success
-   * @throws Exception
-   *
-  public function upgrade_4200() {
-    $this->ctx->log->info('Applying update 4200');
-    CRM_Core_DAO::executeQuery('UPDATE foo SET bar = "whiz"');
-    CRM_Core_DAO::executeQuery('DELETE FROM bang WHERE willy = wonka(2)');
-    return TRUE;
-  } // */
+   * Adds "Case Type Categories" field to the `civicrm_case_type` table
+   */
+  public function upgrade_0002() {
```

Comment:
Can we follow the upgrader process for adding a Civicrm upgrader described here: https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/835092529/Add+a+CiviCRM+Upgrader.

Reply:
Fixed.